extensions/LazyTong/Cube.js

```diff
@@ -0,0 +1,1810 @@
+const t = 'data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjIiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmlld0JveD0iMCAwIDgwIDgwIiB3aWR0aD0iODAiIGhlaWdodD0iODAiPjxzdHlsZT4uYXtmaWxsOiNlNjJ9LmJ7ZmlsbDojNmMwfS5je2ZpbGw6IzYzNn0uZHtmaWxsOiNlZDB9LmV7ZmlsbDojMzkwfTwvc3R5bGU+PHBhdGggY2xhc3M9ImEiIGQ9Im0xMS43IDIyLjNsMjguNi0xMy40djM1LjJsLTI4LjYgMTMuM3oiLz48cGF0aCBjbGFzcz0iYiIgZD0ibTY4LjMgMjIuNmwtMjgtMTMuN3YzNS4ybDI4IDEzLjZ6Ii8+PHBhdGggY2xhc3M9ImMiIGQ9Im0zOS43IDcxLjFsLTI4LTEzLjcgMjguNi0xMy4zIDI4IDEzLjZ6Ii8+PHBhdGggY2xhc3M9ImQiIGQ9Im02OC4zIDIyLjZsLTI4LTEzLjctMjguNiAxMy40IDI4IDEzLjZ6Ii8+PHBhdGggY2xhc3M9ImIiIGQ9Im0xMS43IDIyLjNsMjggMTMuNnYzNS4ybC0yOC0xMy43eiIvPjxwYXRoIGNsYXNzPSJlIiBkPSJtNjguMyAyMi42bC0yOC42IDEzLjN2MzUuMmwyOC42LTEzLjR6Ii8+PC9zdmc+';
+void (window.tempExt = {
+    Extension: class {
+        constructor(e) {
+            console.log("%cLazyTong's Cube\n----------\n       　  　▃▆█▇▄▖\n　 　 　 ▟◤▖　　　◥█▎\n   　 ◢◤　 ▐　　　 　▐▉\n　 ▗◤　　　▂　▗▖　　▕█▎\n　◤　▗▅▖◥▄　▀◣　　█▊\n▐　▕▎◥▖◣◤　　　　◢██\n█◣　◥▅█▀　　　　▐██◤\n▐█▙▂　　     　◢██◤\n◥██◣　　　　◢▄◤\n 　　▀██▅▇▀\n----------\nVer 1.2\nhttps://lt.js.org ", 'color:green;font-weight:bolder');
```

Comment:
Why should you output your extension flags in the console? To publicize your extension? This is highly discouraged. You should only use the console as a platform for outputting uncontrollable errors, and only for debugging errors caused by user usage in partially unpredictable situations.